### PR TITLE
Tf 01 003 features

### PR DESCRIPTION
**Pull request by jobayeructit**
completed feature section.

[features.webm](https://github.com/UCT-IT/edume/assets/149924386/99171e17-9aad-4474-ae31-c6c3ed597ca3)
features

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import HomePage from "./pages/home/HomePage";
 export default function Home() {
   return (
     <main>
-      <HomePage />
+      <HomePage></HomePage>
```
Comment (saminsss) [inline src/app/page.tsx]: Needs to be self closing tag

File: src/components/home/Features/Business/Business.tsx [hunk 2]
```diff
@@ -1,91 +1,63 @@
...
-                </ListItem>
-              )
-            )}
-            <ListItem color="primary" display="flex" gap="16px">
-              <ListIcon as={CheckIcon} color="black" mt="5px" />
-              <Link as={NextLink} href="/">
-                <span>{businessData.businessLink}</span>
+    <div className="container px-[20px] md:pb-16 md:px-0 mx-auto grid grid-cols-1 md:grid-cols-2 gap-5">
+      <div className="p-7 relative">
+        <Image
```
Comment (saminsss) [inline src/components/home/Features/Business/Business.tsx:19]: Self closing tag

File: src/components/home/Features/Clients/Clients.tsx [hunk 3]
```diff
@@ -1,87 +1,63 @@
...
+          <CommonButton
+            href="/"
+            text={data.btn as string}
+            hover="#5AA8EC"
+            color="primary"
+          ></CommonButton>
+        </div>
+      </div>
+      <div className="p-7 relative">
+        <Image
```
Comment (saminsss) [inline src/components/home/Features/Clients/Clients.tsx:54]: Self closing

File: src/components/home/Features/Clients/Clients.tsx [hunk 4]
```diff
@@ -1,87 +1,63 @@
...
-          </Button>
-        </GridItem>
-        <GridItem pos="relative" order={[1, 1, 2]}>
-          <Image alt="client image" src={clientImage}></Image>
-        </GridItem>
-      </Grid>
-    </Container>
+            </li>
+          </ul>
+          <CommonButton
```
Comment (saminsss) [inline src/components/home/Features/Clients/Clients.tsx:45]: Self closing

File: src/components/home/Features/Communication/Communication.tsx [hunk 5]
```diff
@@ -1,91 +1,63 @@
...
-                </ListItem>
-              )
-            )}
-            <ListItem color="primary" display="flex" gap="16px">
-              <ListIcon as={CheckIcon} color="black" mt="5px" />
-              <Link as={NextLink} href="/">
-                <span>{communicationData.communicationLink}</span>
+    <div className="container px-[20px] md:pb-16 md:px-0 mx-auto grid grid-cols-1 md:grid-cols-2 gap-5">
+      <div className="p-7 relative">
+        <Image
```
Comment (saminsss) [inline src/components/home/Features/Communication/Communication.tsx:19]: Self closing

File: src/components/home/Features/Communication/Communication.tsx [hunk 6]
```diff
@@ -1,91 +1,63 @@
...
-            mt={6}
-          >
-            {communicationData.communicationBtn}
-          </Button>
-        </GridItem>
-      </Grid>
-    </Container>
+            </li>
+          </ul>
+          <CommonButton
```
Comment (saminsss) [inline src/components/home/Features/Communication/Communication.tsx:52]: Self closing

File: src/components/home/Features/Feature/Feature.tsx [hunk 7]
```diff
@@ -52,12 +51,17 @@ const Feature = () => {
               </Link>
             </li>
           </ul>
-          <FeatureButton href="/" text={featureData.btn}></FeatureButton>
+          <CommonButton
```
Comment (saminsss) [inline src/components/home/Features/Feature/Feature.tsx:54]: Self closing

File: src/components/home/Features/PaidSection/PaidSection.tsx [hunk 8]
```diff
@@ -1,89 +1,63 @@
...
-            fontWeight="normal"
-          >
-            {getPaidData.getPaidContent?.map((item: string, index: number) => (
-              <ListItem key={index} mb="10px" display="flex" gap="16px">
-                <ListIcon as={CheckIcon} color="black" mt="5px" />
-                {item}
-              </ListItem>
+    <div className="container px-[20px] md:pb-16 md:px-0 mx-auto grid grid-cols-1 md:grid-cols-2 gap-5">
+      <div className="p-7 relative">
+        <Image
```
Comment (saminsss) [inline src/components/home/Features/PaidSection/PaidSection.tsx:19]: Self closing

File: src/components/home/Features/PaidSection/PaidSection.tsx [hunk 9]
```diff
@@ -1,89 +1,63 @@
...
-            mt={6}
-          >
-            {getPaidData.getPaidBtn}
-          </Button>
-        </GridItem>
-      </Grid>
-    </Container>
+            </li>
+          </ul>
+          <CommonButton
```
Comment (saminsss) [inline src/components/home/Features/PaidSection/PaidSection.tsx:52]: Self closing

**Review by saminsss — CHANGES_REQUESTED** [review]
Please remove this [line](https://github.com/UCT-IT/edume/pull/28/files#diff-b6a23ccee71e80a59cf81ed4fc2502c8804182c73c7539ec7f5445b88e89271aL250)

```
bg: {}
```
from locales then you will not need to put in checks for undefined for strings so
```
strings?.features?.business?.title
```
can just be
```
strings.features.business.title
```

File: src/components/home/Features/Business/Business.tsx [hunk 10]
```diff
@@ -1,91 +1,63 @@
...
+        ></Image>
+      </div>
+      <div className="">
+        <div>
+          <h2 className="mb-4 font-medium text-2xl">{data.title}</h2>
+          <ul>
+            {data.content?.map((data, index) => (
+              <li
+                className="text-textColor mb-3 flex text-[16px] font-normal gap-3"
+                key={index}
```
Comment (saminsss) [inline src/components/home/Features/Business/Business.tsx]: Shouldn't use index for keys

**Comment by jobayeructit** [comment]
> Please remove this [line](https://github.com/UCT-IT/edume/pull/28/files#diff-b6a23ccee71e80a59cf81ed4fc2502c8804182c73c7539ec7f5445b88e89271aL250)
> 
> ```
> bg: {}
> ```
> 
> from locales then you will not need to put in checks for undefined for strings so
> 
> ```
> strings?.features?.business?.title
> ```
> 
> can just be
> 
> ```
> strings.features.business.title
> ```

I tried using like you in works.tsx file but its showing 'strings.works' is possibly 'undefined'.ts(18048)
if I use optional chaining then I got no error

**Review by saminsss — APPROVED** [review]

**Pull request merged**